int64[] motor_speeds
int64[] motor_angles
int64[] motor_shovel
